src/main/java/org/folio/inventory/validation/InstancePrecedingSucceedingTitleValidators.java

```diff
@@ -0,0 +1,53 @@
+package org.folio.inventory.validation;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.domain.instances.titles.PrecedingSucceedingTitle;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+import org.folio.inventory.support.http.server.ValidationError;
+
+public final class InstancePrecedingSucceedingTitleValidators {
+
+  private InstancePrecedingSucceedingTitleValidators() {}
+
+  public static CompletableFuture<Instance> refuseWhenUnconnectedHasNoTitle(
+    Instance instance) {
+
+    final ValidationError succeedingError = new ValidationError(
+      "Title is required for unconnected succeeding title", "succeedingTitles.title", null);
+    final ValidationError precedingError = new ValidationError(
+      "Title is required for unconnected preceding title", "precedingTitles.title", null);
+
+    return refuseWhenUnconnectedHasNoTitle(instance, instance.getSucceedingTitles(), succeedingError)
+      .thenCompose(prev -> refuseWhenUnconnectedHasNoTitle(instance, instance.getPrecedingTitles(), precedingError));
+
+  }
```

Comment:
Please could the line lengths be consistent in this method. If the method parameter is being split onto a separate line due to length, that would suggest that at least the second line of the return statement is too long.